Tests/Marketplace/search_and_install_packs.py

```diff
@@ -124,102 +108,51 @@ def create_dependencies_data_structure(response_data: dict, dependants_ids: list
         create_dependencies_data_structure(response_data, next_call_dependants_ids, dependencies_data, checked_packs)
 
 
-def get_pack_dependencies(client: demisto_client, pack_data: dict, lock: Lock):
-    """ Get the pack's required dependencies.
+def get_pack_dependencies(client: demisto_client, pack_id: str, lock: Lock) -> dict | None:
```

Comment:
Make sure we are not using this function in another build flows.

Reply:
Already did, it's not in use anywhere else